packages/terra-status-view/src/StatusView.scss

```diff
@@ -0,0 +1,111 @@
+
+    > svg {
+      background-repeat: no-repeat;
+      background-size: contain;
+      height: 100px;
+      width: 100px;
+    }
+
+    > .no-data {
+      background: var(--terra-status-view-glyph-no-data, inline-svg("<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48' class='is-bidi'><path d='M45.8 12c1.2 0 2.2 1 2.2 2.2v26.7c0 1.2-1 2.2-2.2 2.2H2.2C1 43 0 42 0 40.8V7.2C0 6 1 5 2.2 5H15c1.4.1 2.7.6 3.7 1.6l3.8 3.9c1 1 2.3 1.5 3.7 1.6l19.4-.1h.2z'/></svg>"));
+    }
+
+    > .no-matching-results {
+      background: var(--terra-status-view-glyph-no-matching-results, inline-svg("<svg data-name='Layer 1' xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48'><path d='M47.4 43.2l-13-13A19.66 19.66 0 0 0 38 19a19 19 0 1 0-7.9 15.4l13 13a2.05 2.05 0 0 0 2.9 0l1.4-1.4a1.93 1.93 0 0 0 0-2.8zM3 19a16 16 0 1 1 16 16A16 16 0 0 1 3 19z'/></svg>"));
+    }
+
+    > .not-authorized {
+      background: var(--terra-status-view-glyph-not-authorized, inline-svg("<svg data-name='Layer 1' xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48'><path d='M38.8 20H35v-9a11 11 0 0 0-22 0v9H9.2A2.22 2.22 0 0 0 7 22.2v23.7A2.2 2.2 0 0 0 9.2 48h29.7a2.22 2.22 0 0 0 2.2-2.2V22.2a2.38 2.38 0 0 0-2.3-2.2zM26 33.5V40h-4v-6.5a3.26 3.26 0 0 1-1-2.3 3 3 0 0 1 3-3 3.08 3.08 0 0 1 3 3 3.26 3.26 0 0 1-1 2.3zm6-14.9V20H16v-9a8 8 0 0 1 16 0z'/></svg>"));
+    }
+
+    > .error {
+      background: var(--terra-status-view-glyph-error, inline-svg("<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48'><path fill='#E50000' d='M24 0c13.3 0 24 10.7 24 24S37.3 48 24 48 0 37.3 0 24C0 10.8 10.7 0 23.9 0h.1z'/><path fill='#FFF' d='M7 21.5h34v5H7v-5z'/></svg>"));
+    }
+
+    > .no-service {
+      background: var(--terra-status-view-glyph-no-service, inline-svg("<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48' class='is-bidi'><path d='M19 17.9c-4.1 0-7.8 1.7-10.5 4.3l-3.1-3.1c3.5-3.5 8.3-5.6 13.6-5.6s10.1 2.1 13.6 5.6L29.7 22c-.1 0-.1.1-.2.1-2.8-2.6-6.4-4.2-10.5-4.2zm15.7-.9l3.3-3.3C33.1 9 26.4 6 19 6S4.9 9 0 13.7L3.3 17c4-4 9.6-6.5 15.7-6.5S30.7 13 34.7 17zM21.2 28.9c-.7-.4-1.4-.6-2.2-.6-1.2 0-2.3.5-3.1 1.3l3.1 3.1 1.1-1.1c.3-.9.7-1.8 1.1-2.7zm1.8-2.4c1-1.1 2.2-2.1 3.5-2.9-2-1.6-4.6-2.6-7.4-2.6-3.2 0-6.2 1.3-8.4 3.4l3.1 3.1c1.3-1.3 3.2-2.2 5.2-2.2 1.5 0 2.8.4 4 1.2z'/><path d='M34 24c-6.6 0-12 5.4-12 12s5.4 12 12 12 12-5.4 12-12-5.4-12-12-12zm-9 12c0-5 4-9 9-9 1.9 0 3.7.6 5.2 1.7L26.7 41.2C25.6 39.7 25 37.9 25 36zm9 9c-1.9 0-3.7-.6-5.2-1.7l12.5-12.5c1 1.5 1.7 3.3 1.7 5.2 0 5-4 9-9 9z'/></svg>"));
+    }
+
+    > .custom {
+      background: var(--terra-status-view-glyph-custom, none);
+    }
+  }
+
+  .message-content-group {
+    align-items: center;
+    display: block;
+    flex-direction: column;
+    padding-left: 10px;
+    padding-right: 10px;
+    width: 100%;
+  }
+
+  .title {
+    color: var(--terra-status-view-title-color, #000);
+    flex: 0 0 auto;
+    flex-wrap: wrap;
+    font-family: var(--terra-status-view-title-font-family, 'Roboto');
```

Comment:
Roboto probably isn't the right font for the open source theme. Also, this should just inherit from Base.